ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -70,7 +70,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-openshift-ansible-release-3.11.yaml
-            name: ci-operator-configs
+            name: ci-operator-master-configs
```

Comment:
This looks for a 3.11 config so this should reference `ci-operator-3.x-configs` CM